src/SWP/Bundle/ContentBundle/Resources/config/doctrine-phpcr/Article.phpcr.yml

```diff
@@ -35,6 +35,9 @@ SWP\Bundle\ContentBundle\Doctrine\ODM\PHPCR\Article:
             nullable: true
         isPublishable:
             type: boolean
+        metadata:
```

Comment:
OK, I just noticed that the variable is redeclared in the derived class (which incidentally has the same name as its base class) with a different type. That's pretty funky, I think. You can do it in php, I guess, but whether it's a good idea or not, I wonder.